### PR TITLE
CLI now creates .travis.yml file

### LINKED ISSUE #3: Populate Travis File
As a user, I want the .travis.yml that is generated from hammer-cli to be populated with the industry standard node.js configurations. 

- [ ] Verify that the .travis.yml file is generated with the standard node.js configurations.

### PR DESCRIPTION
**Pull request by jackcmeyer**
closes #3 

* CLI will now create a simple .travis.yml file if the user select TravisCI

### CODE REVIEW

File: src/travis-ci/travis-ci-create.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+const travisCIFileContents = '' +
```
Comment (enclark) [inline src/travis-ci/travis-ci-create.js]: Should we make these template files like the index.js and package.json have?

File: src/travis-ci/travis-ci-create.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    'notifications:\n' +
+    '  email:\n' +
+    '    on_success: never';
+
+export default function initTravisCI(config) {
+  console.log('Creating .travis.yml...');
+
+  fs.writeFile(`${config.projectName}/.travis.yml`, travisCIFileContents, (err) => {
+    if (err) {
+      console.log(`ERROR: Failed to write .travis.yml file\n${err.toString()}`);
```
Comment (enclark) [inline src/travis-ci/travis-ci-create.js]: This is minor, but I've been writing my errors differently.  We should discuss what format we want for them exactly"

File: src/travis-ci/travis-ci-create.js (hunk 1 above)
Comment (jackcmeyer) [reply src/travis-ci/travis-ci-create.js]: I have an issue #21 which we will do this in. The #4 has the same issue.

File: src/travis-ci/travis-ci-create.js (hunk 2 above)
Comment (jackcmeyer) [reply src/travis-ci/travis-ci-create.js]: I agree. Let's talk about that today.

File: src/travis-ci/travis-ci-create.js (hunk 1 above)
Comment (enclark) [reply src/travis-ci/travis-ci-create.js]: Sounds good!

**Pull request merged**